### PR TITLE
Overview Page

### PR DESCRIPTION
**Pull request by felixhummel**
If would be great if there was an overview page like [this](http://felixhummel.github.io/memegen/) but prettier.

pwease

### CODE REVIEW

File: memegen/routes/overview.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from flask import Blueprint, current_app as app, url_for, render_template
+# http://stackoverflow.com/questions/16061514/pylint-pylint-unable-to-import-flask-ext-wtf
+from flask.ext.api.decorators import set_renderers  # pylint: disable=E0611,F0401
```
Comment (jacebrowning) [inline memegen/routes/overview.py]: Or you could just do `from flask_api.decorators import set_renderers`. The `flask.ext` imports are a sort of hack.

**Comment by jacebrowning** [comment]
Maybe add an integration test (`tests/test_routes_overview.py`) that checks a few lines in the generated HTML?

**Comment by felixhummel** [comment]
Doing exactly that atm.
Automated tools beat laziness.
Nice.

File: memegen/routes/overview.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+from flask.ext.api.renderers import HTMLRenderer  # pylint: disable=E0611,F0401
+
+
+blueprint = Blueprint('overview', __name__, url_prefix="/overview")
+
+
+def _urls_generator():
+    for template in sorted(app.template_service.all()):
+        url = url_for("image.get", key=template.key, path="hello-world",
```
Comment (jacebrowning) [inline memegen/routes/overview.py]: If you do `path = template.default or "hello/world"`, you can leverage the default text set in some of the templates.

See: https://github.com/jacebrowning/memegen/blob/3d7a21a1e04a5bee7f4ec8bca576b640b1087e92/memegen/routes/templates.py#L32-34

**Comment by jacebrowning** [comment]
Just throwing out another idea:

It might be cool if the images were links to the raw images and the links below take you to the "links" page.

**Comment by felixhummel** [comment]
This is getting out of hand.

firstworld

**Comment by felixhummel** [comment]
That's it for me for today. :)

BTW: http://knowyourmeme.com/memes/nothing-to-do-here-jet-pack-guy is missing...

**Comment by jacebrowning** [comment]
Thanks, this is great! I'll add a link from the root after merging.

**Comment by jacebrowning** [comment]
Added: http://memegen.link/templates/jetpack

**Pull request merged**